tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35,36,py},py{27,36}-flake8,doclint,commitlint,docstrings
+envlist = py{27,34,35,36,37,38,py},py{27,37}-flake8,doclint,commitlint,docstrings
 minversion = 2.5.0
```

Comment:
This `minversion` setting corresponds with the lower-bound in `dev-requirements.txt`. Please make it correspond with the value in `dev-requirements.txt`. It will help signal to developers that they will need to update their version of tox locally for the configuration to work.

Reply:
https://github.com/sigmavirus24/github3.py/pull/881/files#diff-913fedaed3cb05fe362c0ceb35e2d827R8 is a change in __ dev-requirements.txt__ so that __3.1.3__ is the new minversion.